scripts/db_operations.sh

```
@@ -0,0 +1,215 @@
+    "init")
+      initialize_db
+      ;;
+    "migrate")
+      run_migrations
+      ;;
+    "rollback")
+      if [ -z "$2" ]; then
+        log_message "error" "Version number required for rollback"
+        exit 1
+      fi
+      rollback_migrations $2
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)